src/test/java/com/ncc/aif/ExamplesAndValidationTest.java

```diff
@@ -0,0 +1,1138 @@
+                    presidentObamaIsBarack, 0.5), system, null);
+
+            final Resource secretaryClintonIsBill = markAsPossibleClusterMember(model,
+                    secretaryClinton, billClintonCluster, 0.5, system);
+            final Resource secretaryClintoIsHillary = markAsPossibleClusterMember(model,
+                    secretaryClinton, hillaryClintonCluster, 0.5, system);
+            markEdgesAsMutuallyExclusive(model, ImmutableMap.of(secretaryClintonIsBill, 0.5,
+                    secretaryClintoIsHillary, 0.5), system, null);
+
+            // relation that President Obama (of uncertain reference) worked with Secretary Clinton (of uncertain reference)
+            // is asserted in document 2
+            final Resource relation = makeRelationInEventForm(model, getUri("relation-1"),
+                    ResourceFactory.createResource(SeedlingOntologyMapper.NAMESPACE_STATIC + "PersonalSocial.Business"),
+                    ResourceFactory.createResource(SeedlingOntologyMapper.NAMESPACE_STATIC + "PersonalSocial.Business_Person"),
+                    presidentObama,
+                    ResourceFactory.createResource(SeedlingOntologyMapper.NAMESPACE_STATIC + "PersonalSocial.Business_Person"),
+                    secretaryClinton, getAssertionUri(), system, 0.75);
+
+            // mark justification "President Obama worked with Secretary Clinton"
+            markTextJustification(model, relation, "doc2", 0, 10, system,
+                    0.75);
+
+            dumpAndAssertValid(model, "create a relation where both endpoints are ambiguous (unrestricted way)",
+                    true);
+        }
+
+        @Test
+        void createEntityWithDiskBaseModelAndWriteOut() {
+            final Model model = createDiskBasedModel(true);
+
+            // every AIF needs an object for the system responsible for creating it
+            final Resource system = makeSystemWithURI(model, getTestSystemUri());
+
+            final Resource entity = makeEntity(model, putinDocumentEntityUri, system);
+            final Resource entityIsAPerson = markType(model, getAssertionUri(), entity, SeedlingOntologyMapper.PERSON,
+                    system, 0.5);
+//        final Resource entityIsAPoliticalEntity = markType(model, getAssertionUri(), entity,
+//                SeedlingOntologyMapper.GPE, system, 0.2);
+
+            markTextJustification(model, ImmutableSet.of(entityIsAPerson),
+                    "HC000T6IV", 1029, 1033, system, 0.973);
+
+            Path filename = writeModelToDisk(model);
+
+            final Model model2 = readModelFromDisk(filename, true);
+            Resource rtest = model2.getResource(putinDocumentEntityUri);
+            assertNotNull(rtest, "Entity does not exist");
+        }
+
+    }
+
+
+    /**
+     * DISABLED because they all use ColdStart.
+     * <p>
+     * Don't do what these do!
+     * <p>
+     * These should fail to validate.
+     */
+    @Nested
+    class InvalidExamples {
```

Comment:
We should rewrite these with the seedling ontology. As they are invalid examples, it should be relatively straight-forward.

Reply:
Adding the InvalidExamples was deferred to AIDA-500.

Reply:
I'm a little surprised that this was deferred, but given the scope of 447, I guess that makes sense. I would really like to see these completed before we mark the Kotlin version as deprecated.

Reply:
I added a note to AIDA-500 and AIDA-468 (deprecate ColdStart).